e2e/support/cypress.js

```diff
@@ -66,3 +91,26 @@ Cypress.on("window:load", window => {
     return addEventListener.apply(this, arguments);
   };
 });
+
+// cypress-terminal-report
+afterEach(() => {
+  cy.wait(50, { log: false }).then(() =>
+    cy.addTestContext(Cypress.TerminalReport.getLogs("txt")),
+  );
+});
+
+const options = {
+  collectTypes: [
+    "cons:log",
+    "cons:info",
+    // 'cons:warn', - disabled by intension because of noise from mbql
```

Comment:
```suggestion
    // 'cons:warn', - intentionally disabled because of noise from mbql
```

Reply:
thanks, I need to start correcting comments with chatgpt